.travis.yml

```diff
@@ -13,10 +13,7 @@ python:
   - "3.5"
   - "3.6"
   - "3.7"
-
-before_install:
-  # Workaround Travis environment.
-  - sudo rm /etc/boto.cfg
```

Comment:
This step was failing the build because `/etc/boto.cfg` no longer exists on the build worker file system.